File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        nums = sorted(nums)
        ans = [0]
        for i,num in enumerate(nums):
            if num == 1 + nums[i-1]:
                ans[-1] += 1
            elif i != 0 and num == nums[i-1]:
                continue
            else:
                ans.append(1)
    
        return max(ans)

File: Data Structures & Algorithms/maximum-subarray/submission-2.cpp
class Solution {
public:
    int maxSubArray(vector<int>& nums) {
        int ans = INT_MIN;
        int dp = 0;

        for (int i = 0; i< nums.size(); i++){
            dp = max(dp,0) + nums[i];
            ans = max(ans, dp);
        }
        return ans;
    }
};